tests/Boascan.test.ts

```diff
@@ -721,7 +721,7 @@ describe("Test of Stoa API Server", function () {
             validators: 6,
             frozen_coin: '120000000000000',
             total_reward: '0',
-            circulating_supply: "4999999998096000",
+            circulating_supply: 4999999998096000,
```

Comment:
Why did you convert it into a number without returning it to a string?

Reply:
In IBOAStats interface, the circulating supply is number type